projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -314,7 +314,7 @@
 		word-break: normal;
 		color: unset;
 		opacity: 0.6;
-		font-size: 85%;
+		font-size: 72%;
```

Comment:
Why are we doing this in this PR?

Reply:
Basically because we have a lot of broken stuff when it comes to new selectors and animated and outlined styles. This is just a wee fix for the font size given to notched labels on those form styles.

I'll be adding more like this.

Reply:
Let's get generic fixes merged in separate PRs to isolate them for easier testing.